ubuntu/small-ubuntu-20.04.pkr.hcl

```diff
@@ -74,4 +74,15 @@ build {
     script = "provisioners/cleanup.sh"
   }
 
+  post-processor "manifest" {
```

Comment:
This existed because I wanted to minimise what was uploaded to `ops` VO, but with the new approach for syncing, we can drop it.